pkg/util/hlc/hlc.go

```diff
@@ -99,6 +103,29 @@ type WallClock interface {
 	Now() time.Time
 }
 
+// Logger is used by Clock when clock anamolies are detected.
+type Logger interface {
+	// Warningf is used when the Clock detects non-fatal
+	// conditions.
+	Warningf(ctx context.Context, format string, args ...interface{})
+	// Fatalf is used when forwardClockJump detection is enabled
+	// and the Clock detects a fatal forward clock jump.
+	//
+	// Production implementations of Fatalf should halt the
+	// process in manner appropriate to the caller.
+	Fatalf(ctx context.Context, format string, args ...interface{})
+}
+
+// panicFataler is a Logger that panics when Fatal is called.
+type panicLogger struct{}
+
+var DefaultLogger Logger = &panicLogger{}
```

Comment:
nit: i feel like this var should be named `PanicLogger`. I wouldn't want any production code to accidentally use this logger, instead of the `CRDBLogger`

Reply:
Done.